src/OpenTelemetry.Instrumentation.SqlClient/README.md

```diff
@@ -142,6 +142,11 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
+On .NET Framework, the instrumentation capabilities for System.Data.SqlClient is
+limited. It only only exposes stored procedure names and not the full query
```

Comment:
```suggestion
On .NET Framework, the instrumentation capabilities for `System.Data.SqlClient` is
limited. It only exposes stored procedure names and not the full query
```

Reply:
nit: Consider combining these two sentences to make this easier to read.

example: "On .NET Framework, `System.Data.SqlClient` only exposes stored procuderes names and not the full query."